src/test/ui/nll/outlives-suggestion-simple.stderr

```diff
@@ -48,5 +48,71 @@ help: adding the following bounds may resolve your lifetime errors
    |
    = help: 'a and 'b must be the same; replace one with the other
 
-error: aborting due to 4 previous errors
+error: lifetime may not live long enough
+  --> $DIR/outlives-suggestion-simple.rs:22:5
+   |
+LL | fn foo4<'a, 'b, 'c>(x: &'a usize) -> (&'b usize, &'c usize) {
+   |         --  -- lifetime `'b` defined here
+   |         |
+   |         lifetime `'a` defined here
+...
+LL |     (x, x)
+   |     ^^^^^^ returning this value requires that `'a` must outlive `'b`
+
+help: adding the following bounds may resolve your lifetime errors
+   |
+   = help: 'a: 'b
+
+error: lifetime may not live long enough
+  --> $DIR/outlives-suggestion-simple.rs:31:9
+   |
+LL |     pub fn foo<'a>(x: &'a usize) -> Self {
+   |                -- lifetime `'a` defined here
+LL |         Foo { x }
+   |         ^^^^^^^^^ returning this value requires that `'a` must outlive `'static`
+
+help: adding the following bounds may resolve your lifetime errors
+   |
+   = help: replace 'a with `'static`
```

Comment:
This isnt a bound unlike what the help above suggests (`following bounds`).

Reply:
Updated the messages. Let me know what you think :)